nilearn/tests/test_signal.py

```diff
@@ -103,14 +49,24 @@ def generate_trends(n_features=17, length=41):
 
 
 def generate_signals_plus_trends(n_features=17, n_samples=41):
```

Comment:
Probably a note for a separate refactoring PR but this function exists yet several tests re-implement the lines instead of using it